wordpress-mobile/gutenberg-mobile
# Add support for `"multiple": false` block types

Fixes #1156 

### Related PR
`gutenberg`: https://github.com/WordPress/gutenberg/pull/28339

### Steps to test

1. Open the demo app on either android or ios. The demo already has a **More** block added to the post.
2. Tap **+** to open the block menu - notice the **More** block is disabled. Verify tapping the **More** option does not work.
3. Cancel the block menu. 
4. Delete the **More** block in the post.
5. Tap **+** to open the block menu - notice the **More** block is now enabled. Tap to add this block to the post. Verify success.

PR submission checklist:
